src/reporters/browser.js

```diff
@@ -38,10 +37,15 @@ export default class BrowserReporter {
     `
 
     // Log to the console
-    consoleLogFn(
-      '%c' + [tag, type].filter(Boolean).join(':'),
-      style,
-      ...logObj.args
-    )
+    if (typeof logObj.args[0] === 'string') {
```

Comment:
Should we also check for %c existence in first arg?

Reply:
Having a `%` character doesn't necessarily mean that the string is formatted because it could be used a standalone character. Also I think it adds complexity to the code for a low value. Because even if no `%s` (or other) is present it still works.

Reply:
Okay 😊